libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -489,7 +489,12 @@ bool AP_AHRS_NavEKF::get_position(struct Location &loc) const
     }
 #endif
     }
-    return AP_AHRS_DCM::get_position(loc);
+
+    // fall back to position from DCM
+    if (!always_use_EKF()) {
```

Comment:
shouldn't this be used also in EKFType::NONE: ?

Reply:
@khancyr, thanks for the review.  Yes, the same line, ""AP_AHRS_DCM::get_position(loc)" is in the case statement for EKFType::NONE.

Reply:
@rmackay9 I was speaking about the always_use_EKF check that isn't done in the EKFType::NONE: case

Reply:
OK, I see.  I don't think it is possible for the EKF type to be None if always_use_EKF() is true.  Just looking at the other methods I don't think we ever try to enforce use of an EKF if the EKF type is None.

Reply:
ok good!